java/arcs/android/storage/database/DatabaseImpl.kt

```diff
@@ -1653,6 +1659,11 @@ class DatabaseImpl(
                     counters?.increment(DatabaseCounters.GET_TEXT_VALUE_ID)
                     TABLE_TEXT_PRIMITIVES to value.toString()
                 }
+                PrimitiveType.Instant.id -> {
+                    require(value is Instant) { "Expected value to be a Instant, got $value" }
+                    counters?.increment(DatabaseCounters.GET_TEXT_VALUE_ID)
+                    TABLE_TEXT_PRIMITIVES to value.toString() // TODO: XXXXXXXXXX
```

Comment:
TODO? can you explain what is the strategy here? Ideally every todo as bug associated with it TODO(b/...): "..." (and an indication of what is to be done).

Didn't you say you needed to make these fields queryable, hence you needed to store them as numbers?
I you start with strings, then converting to numeric later will be a nightmare if you need to migrate existing data.
I think a better idea is to hold off storing anything until you have a proper solution. Ie drop fields here for now (and don't attempt to read them). Then in a follow up pr introduce proper persistence of instants.

Reply:
Thanks @galganif, as the todo marks, I hadn't finished looking into the SQL support for bigint and so hadn't updated this to use a numeric.

I'll switch them over and can I get you to take another look at it?

Reply:
Are you switching in this pr? Just let me know if you want me to take another look.